src/test/scala/org/broadinstitute/dsde/vault/services/uBAM/UBamDescribeServiceSpec.scala

```diff
@@ -8,9 +8,9 @@ import spray.http.HttpHeaders.Cookie
 import spray.http.StatusCodes._
 import spray.httpx.SprayJsonSupport._
 
-class UBamDescribeServiceSpec extends VaultFreeSpec with UBamDescribeService with UBamIngestService {
+class UBamDescribeServiceSpec extends VaultFreeSpec with UBamDescribeService with UBamIngestService with UBamDescribeListService{
 
-  override val routes = uBamDescribeRoute
+  override val routes = uBamDescribeRoute;  uBamDescribeListRoute
```

Comment:
Pretty sure that `;  ` was meant to be a ` ~ `. As it is now, those extra characters don't do anything.

**BUT** this touches on an interesting issue. Something feels wrong about these variable names, especially when they collide like this. This line was to fix a compiler error, but the `val` is _never_ used. One could have just as well set the line to `override val routes = null`.

From every example I've seen, the common name for the variable composed of the different routes is `routes`. But if one wants to have a `ServiceSpec` extending multiple `Service`s, then you run into this name collision and compiler error.

Maybe the variable names should be named uniquely in their `trait`, such as `val describeRoute` and `val describeListRoute`? The other option: do what I believe was attempted here, arbitrarily defining in this `ServiceSpec` a definition of the `val routes`, even when it's never actually invoked.

Will leave it to any one or more of @davidangb, @jmthibault79, etc. to decide.